src/app/modules/auth/components/sign-in/sign-in.component.scss

```diff
@@ -0,0 +1,44 @@
+@import 'variables';
```

Comment:
any subimport of a stylesheet should begin with an underscore
consider renaming it to _vars